lib/debug/observatory_debugger.dart

```diff
@@ -505,15 +513,9 @@ class ObservatoryIsolate extends DebugIsolate {
   stepIn() => service.resume(isolateRef.id, step: StepOption.kInto);
   stepOver() => service.resume(isolateRef.id, step: StepOption.kOver);
   stepOut() => service.resume(isolateRef.id, step: StepOption.kOut);
-  stepOverAsyncSuspension() =>
-      service.resume(isolateRef.id, step: StepOption.kOverAsyncSuspension);
-  autoStepOver() {
-    if (suspendedAtAsyncSuspension) {
-      return stepOverAsyncSuspension();
-    } else {
-      return stepOver();
-    }
-  }
+  stepOverAsyncSuspension() => service.resume(isolateRef.id, step: StepOption.kOverAsyncSuspension);
```

Comment:
This project is a healthy mix of the dart style and the flutter style guide. I keep lines at <=80 cols unless that hurts readability.